### PR TITLE
Make a Documentation folder and add a FAQ

### PR DESCRIPTION
**Pull request by denised**
I think we just have to have a basic FAQ as to what is in a model.  Here's an attempt at it.  Made a Documentation folder and moved the Extraction.md there too, to keep the top level directory from getting too cluttered.

### CODE REVIEW

File: Documentation/Solution_Model_FAQ.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ * The CO2 emissions resulting from the technology use (C2).  {Need to clarify where this is positive vs. negative.}
+
+
+## VMAs
+
+The term VMA stands for Variable Meta Analysis.  The idea behind Variable Meta Analysis is to search the available literature for suitable estimates for some specific parameter (such as cost to build a power plant), and record them.   These are always scalar values.  (Sometimes there are VMAs for "growth factors" to allow for variation over time).   A researcher has the option to select from low, high or average of these estimates when doing analysis.
+
+Think of VMA as "parameter of the model".
+
+## RSS vs. Land
```
Comment (johnpaulalex) [inline Documentation/Solution_Model_FAQ.md:29]: I'm gathering that RSS should be RRS?

File: Documentation/Solution_Model_FAQ.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+This file is a quick and dirty introduction / glossary / FAQ to how the solution models work, both in the Excel and python.
+This needs to be extended and improved.  We welcome contributions!
+
+
+## What does a solution model do?
+
+Models calculate the economic cost and greenhouse gas impact of a technology under differing assumptions.  The most important assumption is an estimate of how much the technology is adopted over a time horizon (out to 2050).  Again, the _use_ of the technology is an _input_ to the model, not an output.
+
+The _core inputs_ to a solution are:
+ * For RSS models, a "Total Addressable Market" (TAM) and for Land models a "Total Land Allocation" (TLA).  These express the total demand for the technoogy.  For exectricity generation models, it is something like anticipated need for electricity over time.
```
Comment (johnpaulalex) [inline Documentation/Solution_Model_FAQ.md:12]: s/exectricity/electricity/

File: Documentation/Solution_Model_FAQ.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+## REF vs PDS (TAM/Adoption)
+
+There are actually **two** TAMs and **two** Adoptions:  The "Ref" TAM/Adoption and the "PDS" TAM/Adoption.  The Ref version is the TAM/Adoption assuming that "nothing changes" and the "PDS" TAM/Adoption assumes that the technology has some interesting curve of adoption.  The reason for this is that researchers are typically wanting to learn the delta that a solution can have, so the PDS and REF sub-models are the two that are being compared.
+
+## Scenarios
+
+A scenario is a saved computation that includes all the inputs that went in to it, and the outputs that results from it.   It is a snapshot of 
+the solution with a certain set of assumptions.   In the python code, these are found in the `ac` subdirectory, in json format.
+Each solution includes a set of scenarios that have been designed by Project Drawdown researchers.  In the future, there will be more tools (both in this code and at the UI level) that support browsing/creating/analyzing scenarios.
```
Comment (johnpaulalex) [inline Documentation/Solution_Model_FAQ.md:65]: How do these map, if at all, to the 2 public-facing scenarios on the website ('Drawdown Scenario 1 is roughly in-line with 2˚C temperature rise by 2100, while Drawdown Scenario 2 is roughly in-line with 1.5˚C temperature rise at century’s end.')? Are certain scenario names  (like for Scenario 1 and 2) shared across all solutions?

File: Documentation/Solution_Model_FAQ.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+ * Sources are categorized into 'Base', 'Conservative' and 'Ambitous' cases.  This determination is made by whomever enters the data.  Then, during analysis, the researcher can elect to use average data from one of those specific categories, or the average of all of them (ALL SOURCES).
+
+In python these sources will appear in `tam` or `ad` subfolders of the solutions.
+
+It is also possible to specify a "Custom TAM" and/or "Custom Adoption", which amount to simply specifiying the table of data (in python the custom adoptions are stored in the `ca_*` directories.
+
+There are some other even more complex options available for Adoption.  The "Helper Tables" module/tab is the final step of selecting the actual, used Adoption based on all the choices made on other tabs. ==> This is where the actual REF and PDS Adoption are available.
+
+For a researcher using the solution, all the settings they have access to can be found on the 'Advanced Controls' tab (which in python translates to the settings in a scenario file in the ac subdirectory).  These include chosing from available TAM/Adoption options and any parameters that modify them.
```
Comment (johnpaulalex) [inline Documentation/Solution_Model_FAQ.md:53]: s/chosing/choosing/

Comment (johnpaulalex) [inline Documentation/Solution_Model_FAQ.md:53]: Up above you define an abstraction:
(costs, co2) = solution_model_function(tam/tla, adoption, vmas)

But it sounds like tam/tla and adoption are also derived quantities? Is there a similar model for computing them, e.g.
tam/tla = some_other_abstraction1(more_vmas)
adoption = some_other_abstraction2(more_vmas)
?
Seems like 'advanced controls' might be this abstraction? or are all the computations (the 'complex set of interactions') external and the python just reads various files with hardcoded tam/tla/adoption values?

**Review by johnpaulalex — COMMENTED** [review]
Wow this is really helpful. I was really needing more context/definitions for a lot of terms.

File: Documentation/Solution_Model_FAQ.md (hunk 4 above)
Comment (denised) [reply Documentation/Solution_Model_FAQ.md:53]: Awesome feedback @johnpaulalex.  I will fix things presently.    You are write that there is roughly "two stages" of computation ---> VMAs+modeling choices. --> (tam/tla/adoption), then other VMAS + (tam/tla/adoption) --> outputs.
And yes, the VMA's and modeling choices are found on Advanced Controls.
One thing I should add:  if you look at a ScenarioRecord tab in the Excel, the first level of inputs are all on the left, and the computed (tam/tla/adoption) and outputs are on the right.

File: Documentation/Solution_Model_FAQ.md (hunk 3 above)
Comment (denised) [reply Documentation/Solution_Model_FAQ.md:70]: I added some text to address this question --- thank you for bringing it up :-)

Comment (glass-wing) [reply Documentation/Solution_Model_FAQ.md:70]: I found this statement confusing "There are actually **two** TAMs and **two** Adoptions:  The "Ref" TAM/Adoption and the "PDS" TAM/Adoption."
Seems like we should have four in total?

**Pull request merged**